express-server/app.js

```diff
@@ -12,10 +12,18 @@ app.use(express.json())
 app.use(morgan("tiny"))
 app.use(cors())
 
+// fetches best recipes given user food items from API
 app.get('/apirecipes/:ingredients', async (request, response) => {
-    const recipes_url = `https://api.spoonacular.com/recipes/findByIngredients?ingredients=${request.params.ingredients}&number=20&apiKey=${api_key}`;
+    let recipes_url = `https://api.spoonacular.com/recipes/findByIngredients?ingredients=${request.params.ingredients}&number=20&instructionsRequired=true&fillIngredients=true&apiKey=${api_key}`;
```

Comment:
This may be easier to work with if you use a params object: https://masteringjs.io/tutorials/axios/get-query-params